docs/hacking_howto.rst

```diff
@@ -90,6 +90,8 @@ These are required for the minimum installation:
 * libxslt and headers
 * libjpeg and headers
 * zlib and headers
+* libssl library
+* libmysqlclient library
```

Comment:
@safwanrahman: when @willkg said "This line is covered by the MariaDB 5.5 server and client headers line." he means that `libmysqlclient` is "MariaD 5.5 client headers". You are just listing the explicit package name for something we already have in the list. Since this is a generic list of dependencie, not a list of packages to install, this line isn't appropriate. It would be better put in an Ubuntu-specific install page.